scripts/maintenance/migrate_project/src/cli.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+
+import typer
+from db import (
+    db_connection,
+    get_project_and_files_to_migrate,
+    insert_file_meta_data,
+    insert_projects,
+)
+from models import Settings
+from r_clone import assemble_config_file, sync_file
+
+
+def main(config: Path):
+    assert config.exists()  # nosec
```

Comment:
TIP: add instead ``typer`` options to provide guarantees on the file.  SEE https://typer.tiangolo.com/tutorial/parameter-types/path/#path-validations